utils/moduleVisitor.js

```diff
@@ -80,6 +80,19 @@ exports.default = function visitModules(visitor, options) {
     }
   }
 
+  function checkRequireResolve(call) {
+    if (call.callee.type !== 'MemberExpression') return
+    if (call.callee.object.name !== 'require') return
+    if (call.callee.property.name !== 'resolve') return
+    if (call.arguments.length !== 1) return
```

Comment:
i think this should be removed; we should check the first argument in every case that i'm aware of.

Reply:
I re-used the code from `require` for `require.resolve` so the same checks happen for both. They should both be used the same way, shouldn't they?

Reply:
i'm not sure - i suspect `require` might be used differently in requireJS, but `require.resolve`, i think, only exists in CJS.

Reply:
Maybe, for now, `require.resolve` should work only if `commonjs` and `requireResolve` are both set? That way if a different version of it is added to requireJS, we can handle that too.

Reply:
Maybe, for now, `require.resolve` should work only if `commonjs` and `requireResolve` are both set? That way if a different version of it is added to requireJS, we can handle that too.

Reply:
Maybe, for now, `require.resolve` should work only if `commonjs` and `requireResolve` are both set? That way if a different version of it is added to requireJS, we can handle that too.

Reply:
Maybe, for now, `require.resolve` should work only if `commonjs` and `requireResolve` are both set? That way if a different version of it is added to requireJS, we can handle that combination differently.

Reply:
Maybe, for now, `require.resolve` should work only if `commonjs` and `requireResolve` are both set? That way if a different version of it is added to requireJS, we can handle that combination differently.

Reply:
Maybe, for now, `require.resolve` should work only if `commonjs` and `requireResolve` are both set? That way if a different version of it is added to requireJS, we can handle that combination differently.

Reply:
Maybe, for now, `require.resolve` should work only if `commonjs` and `requireResolve` are both set? That way if a different version of it is added to requireJS, we can handle that combination differently.

Reply:
I think this is the best option:

`requireResolve: boolean | { commonjs: ?boolean, amd: ?boolean }`

Reply:
That seems reasonable (altho resolution like that is unlikely to ever show up in AMD)